object/types.proto

```diff
@@ -117,6 +117,12 @@ message Header {
   //   Marks smaller parts of a split bigger object
   // * __NEOFS__EXPIRATION_EPOCH \
   //   Tells GC to delete object after that epoch
+  // * __NEOFS__TICK_TOPIC \
+  //   UTF-8 string topic ID that is used for object notification
+  // * __NEOFS__TICK_EPOCH \
+  //   Base10 encoded number that defines what epoch must produce
+  //   object notification (`0` value produces notification right
+  //   after object put)
```

Comment:
I'd also add a note that the notification body will contain object address in UFT-8 string.

Reply:
thought such details should be in spec, but added

Reply:
NeoFS Spec is automatically generated from this API definitions. =)

Reply:
i meant a few manually written words about notifications